docs/installation-places.md

```diff
@@ -1,35 +0,0 @@
-# Installation
```

Comment:
I deleted the places installation because it seemed identical? I can add a separate section to installation about installing places as well.